doc/howtos/bluetoothspeaker.rst

```diff
@@ -0,0 +1,136 @@
+Bluetooth Speaker
+#################
+
+How to setup your intel-iot-refkit to function as a bluetooth speaker
+=====================================================================
+
+For bluetooth speaker setup you should use the "gateway" image and
```

Comment:
yes, tried to fix but some native english speaker should read the whole thing through, agreed